tgnnexplainer/xgraph/method/subgraphx_tg.py

```diff
@@ -363,10 +375,10 @@ def _initialize_tree(self):
         self.state_map = {self.root_key: self.root}
 
         max_event_idx = max(self.root.coalition)
-        self.curr_t = self.events['ts'][self.events.e_idx==max_event_idx].values[0]
+        self.curr_t = self.timesteps[self.event_indices==max_event_idx][0]
 
     def _node_key(self, coalition):
-        return "_".join(map(lambda x: str(x), sorted(coalition) ) ) # NOTE: have sorted
+        return "_".join(map(lambda x: str(x), np.sort(coalition) ) ) # NOTE: have sorted
```

Comment:
Is this faster?

Reply:
Depends. NumPy's strengths shine the most with big data. With a small coalition like this, it might instead be a bottleneck. I meant to test before pushing but I forgot. Don't really see a difference altogether.